fundamentos/bloco-03-introducao-a-html-e-css/dia-4-html-css-html-semantico/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="pt">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ranking</title>
+    <style>
+      * {
+        text-align: center;
```

Comment:
Fala Lucas, evite utilizar o seletor universal *{ text-align: center}, quando for uma coisa específica. Não que esteja errado, até porque esse seletor tem a especificidade mais baixa, e todos os outros sobreescrevem a ele. 

Mas até pela questão de organização, você vincula o filho ao pai, então o melhor seria colocar o text-align:center como propriedade do elemento ul {text-alig:center}, como a relação é direta, de pai e filho, quando você quiser modificar algo em um arquivo mais extenso, fica mais fácil você alterar quando necessário. 

a utilização do seletor universal você utiliza mais pra dar um pre-set no html, e não é isso que queremos, só queremos posicionar um elemento no meio da página.